Gopkg.lock

```diff
@@ -3,80 +3,106 @@
 
 [[projects]]
   branch = "master"
+  digest = "1:d5852e9d2b4e8a44389e0768abcdb080b5c63e3d5f78c56d77d2513769835375"
   name = "github.com/dustin/go-heatmap"
   packages = [
     ".",
-    "schemes"
+    "schemes",
   ]
+  pruneopts = "UT"
   revision = "b89dbd73785a2348b5e0558f719f27a826987ceb"
 
 [[projects]]
+  digest = "1:ae5d8cbf9034fc6d50b2d42472907d018911dd3bff870e190a757b6394d5ff24"
   name = "github.com/gogo/protobuf"
   packages = [
     "proto",
-    "protoc-gen-gogo/descriptor"
+    "protoc-gen-gogo/descriptor",
   ]
+  pruneopts = "UT"
   revision = "636bf0302bc95575d69441b25a2603156ffdddf1"
   version = "v1.1.1"
 
 [[projects]]
   branch = "master"
+  digest = "1:62c57507df491b657e9ae2645f30958d9964c8eeb380600469eedc951ebb3a0e"
   name = "github.com/golang/freetype"
   packages = [
     "raster",
-    "truetype"
+    "truetype",
   ]
+  pruneopts = "UT"
   revision = "e2365dfdc4a05e4b8299a783240d4a7d5a65d4e4"
 
 [[projects]]
   branch = "convex-hull"
+  digest = "1:c80d112da67920ce049fcf0ec63863b2ef5c1a5c7d4ef0cde8e75292681f038e"
   name = "github.com/golang/geo"
   packages = [
     "r1",
     "r2",
     "r3",
     "s1",
-    "s2"
+    "s2",
   ]
+  pruneopts = "UT"
   revision = "12889707c62e2a5059e345e30450c7e79c70a6fc"
   source = "github.com/markus-wa/geo"
 
 [[projects]]
   branch = "master"
+  digest = "1:ddfe7e1a02102d9b0e10190bf67a4c127f25ec587d5cb6ed7a403a984b7af27a"
   name = "github.com/llgcode/draw2d"
   packages = [
     ".",
     "draw2dbase",
-    "draw2dimg"
+    "draw2dimg",
   ]
+  pruneopts = "UT"
   revision = "274031cf2abe8acd15c4a911e1f7f1295fc4830c"
 
 [[projects]]
+  digest = "1:3e50e9c6b428e43730caea637bc087db0eed3fd1a8610407eb32ed8d2af0fae6"
   name = "github.com/markus-wa/gobitread"
   packages = ["."]
+  pruneopts = "UT"
   revision = "062d24bd67896096b650947bc03b468f769ab665"
   version = "v0.2.1"
 
 [[projects]]
+  digest = "1:5210781e60f69f9eb5c258267ad8295ff186524ece6fcb33715c1a7c7ba1ec88"
   name = "github.com/markus-wa/godispatch"
   packages = ["."]
+  pruneopts = "UT"
   revision = "3c2bfa80b4fa8e97f461b579dda6d698aee02430"
   version = "v1.1.0"
 
 [[projects]]
   branch = "master"
+  digest = "1:c46a80735d879b1b808686638e0a5c0a356ed4edc0981971dab4dad93d8c20cc"
   name = "golang.org/x/image"
   packages = [
     "draw",
     "font",
     "math/f64",
-    "math/fixed"
+    "math/fixed",
   ]
+  pruneopts = "UT"
   revision = "c73c2afc3b812cdd6385de5a50616511c4a3d458"
 
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "cda22bd3cd5cfcaa6e9fab1e1f64ed6670859d06ea62922ac4cb42f7e4e701f7"
+  input-imports = [
```

Comment:
Do you know what happened to the Gopkg.lock?

New dep version that I wasn't using yet maybe?

Reply:
I honestly don't know - this is the first that I've ever used it :slightly_smiling_face: